src/ui/public/courier/fetch/request/request.js

```diff
@@ -1,5 +1,6 @@
 import _ from 'lodash';
 import moment from 'moment';
+import { race } from 'bluebird';
```

Comment:
Woah, I totally forgot that this was an angular context bluebird isn't okay here anyway.. I'll add a Promise.race function to `src/ui/public/promises/promises.js`